VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -346,11 +356,17 @@ protected void onException(Exception ex) {
                 nextPage = 1;
             }
         };
-        getThreadListTask.setProgressCallback(null);
+        // Initially we need to show the spinner at the center of the screen. After that the
```

Comment:
There should be a comma after 'After that'. Applies to the identical comment in `CourseDiscussionPostsSearchFragment` too.